localStorage_value_checker_sample_app/app.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  return {
+    events: {
+      // Framework Events
+      'app.created':'checkLocalStorage', // Call 'checkLocalStorage' function below
+      // Request Events (None)
+      // DOM events
+      'keyup #inputValueId': function(event){ // User clicked the 'enter'/'return' button so store value in localStorage
+        if(event.keyCode === 13)
+          return this.processInputValue();
+      },
+      'click .reset': 'resetValue' // Call 'resetValue' function below
+    },
+    checkLocalStorage: function () { // Look in localStorage for if value is present for 'exampleString'
+        if (this.store('exampleString') === null) { // If no value
```

Comment:
typically when you need a comment to explain what an expression means, it's a sign that you should pull it out into a variable with a name that communicates that meaning (in this example, something like `var noStoredValue = this.store('exampleString') === null;`.

Or if you feel that `this.store('exampleString') === null` communicates that just fine, remove the comment ;)